lib/phail_web/live/phail.ex

```diff
@@ -91,11 +91,10 @@ defmodule PhailWeb.Live.Phail do
   end
 
   defp change_labels(conversation_id, old_label, new_label) do
+    conversation = Conversation.get(conversation_id)
```

Comment:
you can pattern-match this to a conversation in case `.get` returns nil.

Reply:
I'm not sure what I'd do in nil case, so I'm happy to let it crash for the moment. Is there a pattern to signal errors from liveview events.

Reply:
amusingly i have yet to write any liveview beyond our profiling for that liveview PR and testing for the new feature we're working on. XD so i'm not sure. my guess would be following the usual elixir convention of 2-item tuples. so pattern-matching on `{:ok, thing}` and `{:error, error}` to do something.